doc/tutorials/active_matter/active_matter.ipynb

```diff
@@ -210,549 +837,189 @@
-    "Once again, you will have to complete the excercises to obtain a functioning\n",
-    "script. The structure of the blocks and their content should by now be\n",
-    "straightforward for you to understand on the basis of the previous tutorials\n",
-    "and the information provided here. We will therefore focus on the use of this\n",
-    "script.\n",
-    "\n",
-    "<a id='fig:flow_field'></a>\n",
-    "<figure>",
-    "<img src=\"figures/flow_field.svg\" style=\"float: center; width: 40%\"/>\n",
-    "<center>\n",
-    "<figcaption>Fig. 7: (a) Snapshot of the flow field around a pusher particle visualized with ParaView.</figcaption>\n",
-    "</center>\n",
-    "</figure>\n",
-    "\n",
-    "First run the simulation for a puller particle and a position of 0.0. This will\n",
-    "generate output in the directory that you have set up. Examine the content of\n",
-    "this folder and, in particular, the <tt>trajectory.dat</tt> file. From the final\n",
-    "line of this file, you can determine the position of the swimmer at the end of\n",
-    "the run. Now rerun the script with a modified position value, such that the\n",
-    "particle ends up in the center of the box. This generates a second directory.\n",
-    "Now go into this directory and open ParaView. Using the techniques you have\n",
-    "learned in the above tutorial, you can visualize the particle, and using the\n",
-    "methods you have picked up in the Electrokinetics tutorial, you can visualize\n",
-    "the fluid flow around this particle using, *e.g.*, stream lines, a\n",
-    "slice, or arrows. When you are done, the result could look like\n",
-    "<a href='#fig:flow_field'>Fig. 7</a>."
+    "system.integrator.run(HYDRO_PARAMS['run_steps'])"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "## Concluding Remarks\n",
-    "\n",
-    "With that, you have come to the end of this tutorial. We hope you found it\n",
-    "informative and that you have a sufficient understanding of the way to deal\n",
-    "with active matter in **ESPResSo** to set up simulations on your own."
+    "vels = np.squeeze(lbf[:, int(system.box_l[1]/2), :].velocity)\n",
+    "vel_abs = np.linalg.norm(vels, axis=2)\n",
+    "\n",
+    "lb_shape = lbf.shape\n",
+    "xs, zs = np.meshgrid(np.linspace(0.5, box_l[0] - 0.5, num=lb_shape[0]),\n",
+    "                     np.linspace(0.5, box_l[2] - 0.5, num=lb_shape[2]))\n",
+    "\n",
+    "fig_vels, ax_vels = plt.subplots()\n",
+    "im = plt.pcolormesh(vel_abs.T, cmap='YlOrRd')\n",
+    "plt.quiver(xs, zs, vels[:, :, 0].T, vels[:, :, 2].T, angles='xy', scale=0.005)\n",
+    "circ = plt.Circle(particle.pos_folded[[0, 2]], 0.5, color='blue')\n",
+    "ax_vels.add_patch(circ)\n",
+    "ax_vels.set_aspect('equal')\n",
+    "plt.xlabel('x')\n",
+    "plt.ylabel('z')\n",
+    "cb = plt.colorbar(im, label=r'$|v_{fluid}|$')\n",
```

Comment:
@jhossbach please do `make tutorial_am_html` and look at this particular plot. This type of issue cannot be detected in CI because we only generate the html files on the python branch, i.e. after a PR is merged.

Reply:
@jngrad You're right, there was a problem with the `\text` part, changing it to `\mathrm` solved it.